dynalab_cli/test.py

```diff
@@ -33,6 +36,22 @@ def __init__(self, args):
         self.args = args
         self.config_handler = SetupConfigHandler(args.name)
 
+    def use_gpu(self, config) -> bool:
+        task = config["task"]
+        # TODO: should this be stored in the DB ? And exposed in the /tasks API ?
```

Comment:
There's no db in dynalab. Can write a task config in the e.g. common.py file instead